prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml

```diff
@@ -41,6 +41,24 @@ presubmits:
         command:
         - "/go/main.sh"
         args: ["cli_tools/"]
+  - name: cli-tools-presubmit-module-tests
+    cluster: gcp-guest
+    skip_report: true
+    always_run: false
+    run_if_changed: 'cli_tools/.*'
+    trigger: "(?m)^/cli-tools-module-tests$"
+    rerun_command: "/cli-tools-module-tests"
+    context: prow/presubmit/module-tests
+    decorate: true
+    labels:
+      preset-daisy: "true"
```

Comment:
the gotest container doesn't need the daisy credentials, does it?

Reply:
The tests in `cli_tools_tests/module/` execute using the go test framework, but use the credentials while they run.

Reply:
per our offline discussion, once you've done testing, you should add a comment here explaining that this is an integ test and needs creds, and add any other necessary environment variables to the job spec